src/rsz/src/EstimateWireParasitics.cc

```diff
@@ -272,13 +273,20 @@ void Resizer::ensureParasitics()
 }
 
 void Resizer::estimateParasitics(ParasiticsSrc src)
+{
+  std::map<Corner*, std::ostream*> spef_files;
+  estimateParasitics(src, spef_files);
+}
+
+void Resizer::estimateParasitics(ParasiticsSrc src,
```

Comment:
You can initialize the spef_writer_ here and pass it to the functions, instead of passing the map.